tests/virt_autotest/virt_utils.pm

```diff
@@ -119,7 +119,7 @@ sub handle_sp_in_settings {
     # We need small case variable value
     my $var_value = lc(get_required_var("$var_name"));
     # Add $value_for_sp after release for products like sle15, sle16
-    if ($var_value !~ /sp/ && "$value_for_sp") {
+    if ($var_value !~ /sp/ && $var_value !~ /fcs/ && "$value_for_sp") {
```

Comment:
Why not to put both parts in single regexp? E.g.
```perl
$var_value !~ /sp|fcs/
```
We also don't really need wrapping `$value_for_sp` with quotes.

Reply:
yes, will use that instead